setup.py

```diff
@@ -60,7 +60,7 @@ def get_version() -> str:
             "flake8",
             "pytest",
             "pytest-cov",
-            "ray>=0.8.0",
+            "ray>=1.0.0",
```

Comment:
I think ray should be an optional requirement, because it it not necessary at all to use most of the features of Tianshou.

Reply:
Yes, this is optional now since the requirements are under "dev".

Reply:
Indeed my bad, I missed it !